q2_feature_classifier/tests/test_classifier.py

```diff
@@ -223,3 +223,16 @@ def test_unassigned_taxa(self):
             id_ = seq.metadata['id']
             assert ref[id_].startswith(classified[id_]) or \
                 classified[id_] == 'Unassigned'
+
+    def test_autotune_reads_per_batch(self):
+        self.assertEqual(
```

Comment:
Could you add tests including a few different values of `n_jobs` in here, ideally including one that would catch the `int(ceil())` versus `int` case I mentioned above?

Reply:
Also, what happens if `n_jobs` is greater than the number of reads? That should be covered in a test (`ceil` might handle that correctly - I think we'd want the reads per batch to be 1 in this edge case).